helphours/logger.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+
+
+class Logger():
+
+    # The various logging levels to mask out log entries.
+    # e.g. Using LEVEL_WARNING will mask out all DEBUG and INFO messages.
+    LEVEL_DEBUG = 10
+    LEVEL_INFO = 20
+    LEVEL_WARNING = 30
+    LEVEL_ERROR = 40
+    LEVEL_FATAL = 50
+
+    DEBUG_PREFIX = "[Debug]"
+    INFO_PREFIX = "[ Info]"
```

Comment:
Extra space?

Reply:
Those were intentional, it's so they all take up the same width. For example:
```
[Aug 16, 2020][15:21:50][Debug]: Example message
[Aug 16, 2020][15:20:12][ Info]: Example message
[Aug 16, 2020][15:21:50][ WARN]: Example message
[Aug 16, 2020][15:21:50][ERROR]: Example message
[Aug 16, 2020][15:21:50][FATAL]: Example message
```
I feel like it makes it look nicer in a file.

Reply:
But we could change it so the space is after the text, e.g. `[Info ]` if that looks better

Reply:
Nah, looks good!